doc/source/development/test_writing.rst

```diff
@@ -0,0 +1,152 @@
+.. _test_organization:
+
+Test organization
+=================
+Ideally, there should be one, and only one, obvious place for a test to reside.
+Until we reach that ideal, these are some rules of thumb for where a test should
+be located.
+
+1. Does your test depend only on code in ``pd._libs.tslibs``?
+This test likely belongs in one of:
+
+    - tests.tslibs
```

Comment:
This also needs to be aligned with the "Does .." from the first line of this bullet point (the doc link I gave above also has an example of nested lists)